web/app/view/map/MapMarkerController.js

```diff
@@ -188,6 +188,39 @@ Ext.define('Traccar.view.map.MapMarkerController', {
         }
     },
 
+    animateMarker: function (marker, geometry, course) {
+        var start, end, duration, timeout, line, updatePos;
+
+        start = marker.getGeometry().getCoordinates();
+        end = geometry.getCoordinates();
+        line = new ol.geom.LineString([start, end]);
+        duration = Traccar.Style.mapAnimateMarkerDuration;
+        timeout = Traccar.Style.mapAnimateMarkerTimeout;
+
+        updatePos = function (pos, m) {
```

Comment:
Please use full words for names. Pos should be Positions. M should be Marker. etc